R/Accelerated_Failure_time_model.qmd

```diff
@@ -0,0 +1,117 @@
+
+Mathematically, the AFT model can be expressed as:
+
+$\log(T) = X\beta + \varepsilon$
+
+Where:
+
+-   T is the survival time
+
+-   log(T) is the logarithm of the survival time
+
+-   X is a matrix of predictor variables
+
+-   β is a vector of coefficients representing the effects of the predictor variables on the logarithm of the survival time
+
+-   ε is the error term assumed to follow a certain distribution, such as normal, exponential, weibull or log-normal
+
+## Example of AFT model using "Log-Normal Distribution"
+
+```{r}
+library(survival)
+attach(lung)
+# Fit an AFT model using lognormal distribution
+model_aft <- survreg(Surv(time, status) ~ age + sex + ph.ecog, data = lung, dist = "lognormal")
+# Model summary
+summary(model_aft)
+```
+
+The summary output will provide the estimated coefficients, standard errors, and p-values for each predictor variable.
+
+## Hazard Ratio Calculation
+
+```{r}
+# Compute hazard ratios (exponentiated coefficients)
+hazard_ratios <- exp(coef(model_aft))
+hazard_ratios
+```
+
+## Interpretation
+
+-   For age, hazard ratio $< 1$ indicates that for each one-unit increase in age, the survival time is slowed down by a factor of 0.98 (or a 2% decrease).
+-   For sex, hazard ratio $> 1$ indicates that males have 68% higher hazard rate or accelerated survival time.
+-   A hazard ratio of ph.ecog $< 1$ suggests a 30% lower hazard rate or decelerated survival time associated with ph.ecog.
+
+## Example of AFT model using "Weibull Distribution"
+
+```{r}
+attach(ovarian)
+
+# Fit an AFT model using lognormal distribution
+model_aft_wb <- survreg(Surv(futime, fustat) ~ age + resid.ds + rx, data = ovarian, dist = "weibull")
+# Model summary
+summary(model_aft_wb)
+```
+
+## Hazard Ratio Calculation
+
+```{r}
+# Compute hazard ratios (exponentiated coefficients)
+hazard_ratio <- exp(coef(model_aft_wb))
+hazard_ratio
+```
+
+## Interpretation
+
+-   For age, a hazard ratio of 0.93 indicates that for each one-unit increase in age, the survival time is decelerated by a factor of 0.93(or a 7% decrease)
+
+-   For residual disease status, a hazard ratio of 0.60 suggests that a decrease in residual disease status is associated with a 40% lower hazard rate or decelerated survival time.
+
+-   a hazard ratio of 1.67 suggests a 67% higher hazard rate or accelerated survival time for patients receiving a different type of radiation therapy (rx = 2) compared to the reference group (rx = 1).
+
+## Comparison between Cox PH Model & AFT Model with Example
+
+To compare Cox Proportional Hazards(PH) models & Accelerated Failure Time(AFT) models for a lung dataset following steps are performed.
+
+```{r}
+#Fit the Cox PH Hazard Model
+cox_model <- coxph(Surv(time,status)~ age + sex + ph.ecog, data = lung)
+summary(cox_model)
+
+# Fit an AFT model using lognormal distribution
+aft_model <- survreg(Surv(time, status) ~ age + sex + ph.ecog, data = lung, dist = "lognormal")
+summary(aft_model)
+
+#Compare AIC Values
```

Comment:
The AIC is based on the likelihood function, so comparison can be problematic between the semi-parametric Cox (partial likelihood) and parametric AFT models (full likelihood). As mentioned above, I believe focusing on the assumptions each of the models makes may be more valuable.

Reply:
removing for now